setup.py

```diff
@@ -20,8 +20,10 @@
 
 requirements = [
     "networkx>=2.6,<3.0",
-    "openpyxl>=3.0.3,<4.0",
-    "pandas>=1.3.4,<2.0",
+    "openpyxl>=3.0.7,<4.0",
+    "modin[dask]>=0.16.2,<1.0",
+    "bokeh==2.4.3",
```

Comment:
You removed it from PipFile, but not setup.py

Reply:
Thanks! That was very careless of me 😅